Backend/DynamicQR/DynamicQR.Infrastructure/Services/QrCodeRepositoryService.cs

```diff
@@ -23,7 +24,10 @@ public async Task<bool> CreateAsync(string organizationId, QrCode qrCode, Cancel
 
         Azure.Response response = await _tableClient.AddEntityAsync(qrCodeData, cancellationToken);
 
-        return !response.IsError;
+        if (response.IsError)
+            throw new StorageException(response.ReasonPhrase);
+
+        return true;
```

Comment:
Top! Nu is de StorageException toegevoegd.
In dit geval zal het return statement weg kunnen en CreateAsync zal dan een Task teruggeven. Zou je dat nog willen toevoegen @jensdenbraber ?